Automattic/pocket-casts-ios
# Reimagine Sharing: Add sharing to Instagram Stories

| 📘 Part of: #1910 |
|:---:|

Adds sharing for episode & podcast assets to Instagram Stories. Note that this shared asset isn't final (the scaling is a bit off) but this PR is just for sharing to Instagram.

https://github.com/user-attachments/assets/c8d9b99b-9b09-4415-aa28-3913530a22de

## To test

> [!NOTE]
> Enable the `newSharing` feature flag to test this

You may need to update your secrets using `fastlane run configure_update`

1. Play an episode on a device which has Instagram installed
2. Tap on the Share option in the action shelf
3. Tap the "Episode" or "Podcast" sharing option
4. Tap on the "Stories" button in the bottom
5. Ensure that the asset is shared to Instagram stories (I recommend sharing to "Close Friends" so you don't blast this unfinished asset out to everyone)

## Checklist
